genesis/process/shardGenesisBlockCreator.go

```diff
@@ -472,6 +473,7 @@ func createProcessorsForShardGenesisBlock(arg ArgsGenesisBlockCreator, enableEpo
 	disabledBlockTracker := &disabled.BlockTracker{}
 	disabledBlockSizeComputationHandler := &disabled.BlockSizeComputationHandler{}
 	disabledBalanceComputationHandler := &disabled.BalanceComputationHandler{}
+	disabledEpochNotifier := &disabled.EpochNotifier{}
```

Comment:
you should have normal epoch notifier here. as this is called in hardfork as well.

Reply:
done